### PR TITLE
Adapted makefile for linux environment JU

### PR DESCRIPTION
**Pull request by voisardf**
Title says it all - some adaptions of the Makefile

### CODE REVIEW

File: CONST_Makefile [hunk 1]
```diff
@@ -242,9 +244,9 @@ print: .build/print.timestamp
 .build/print.timestamp: $(PRINT_OUTPUT)/$(PRINT_WAR) $(PRINT_REQUIREMENT)
```
Comment (kalbermattenm) [inline CONST_Makefile]: What about the modifications we just made together ?

**Comment by voisardf** [comment]
sorry, missed to push it 🙈
it's done now; thanks!

**Comment by kalbermattenm** [comment]
OK, thanks !

And Jura has tested that as well ?

If yes, merge

**Comment by voisardf** [comment]
that version not yet

**Comment by voisardf** [comment]
got an ok from @sitj-raphael ; merging

**Pull request merged**